plugins/node/opentelemetry-instrumentation-mysql/package.json

```diff
@@ -55,16 +56,17 @@
     "@types/sinon": "10.0.13",
     "gts": "3.1.0",
     "mocha": "7.2.0",
-    "mysql": "2.18.1",
     "nyc": "15.1.0",
     "rimraf": "3.0.2",
     "sinon": "14.0.0",
     "ts-mocha": "10.0.0",
     "typescript": "4.3.5"
   },
   "dependencies": {
+    "@opentelemetry/api-metrics": "^0.32.0",
```

Comment:
When using  `"@opentelemetry/sdk-metrics": "^1.8.0"` as suggested above this can be removed. Of course, the instrumentation needs to import metrics from `@opentelemetry/api` instead in that case.
```suggestion
```

Reply:
done